tests/test_triscord.py

```diff
@@ -11,12 +11,12 @@
 
 import triscord as unit
 import triscord.persistence as persistence
+from triscord import trello
 
-from test_trello import api_actions
-
-_ = api_actions
+from test_trello import api_actions  # pylint: disable=W0611
 
 
+@pytest.mark.usefixture("api_actions")
```

Comment:
Typo: `usefixtures`, https://docs.pytest.org/en/latest/fixture.html?highlight=usefixture#usefixtures

Reply:
When you mark a fixture to be used that way, it becomes redundant to have that fixture as an argument of the test function.